client/src/components/participant-details/psi-cohort.js

```diff
@@ -138,11 +139,29 @@ const PSIRouteTabs = ({
 export const PSICohortView = ({ psiList = [], assignAction, participant, fetchData }) => {
   const match = useRouteMatch();
   const { tab } = useParams();
-  const tabKey = Object.keys(TabDetails).reduce((incoming, key) =>
-    TabDetails[key]?.path === `/${tab}` ? key : incoming
-  );
+  const { auth } = AuthContext.useAuth();
+  const roles = useMemo(() => auth.user?.roles || [], [auth.user?.roles]);
   const baseUrl = match.url.split(tab)[0];
   const sortedList = sortPSI({ psiList, cohort: participant ? participant.cohort : {} });
+
+  // Determine the tags depending on user role
+  const assignCohort = {
+    label: 'Assign Cohort',
+    path: '/assign-cohort',
+    default: true,
```

Comment:
I'm not a huge fan of `default` here, reading this I would think `default` controls the initially selected tab, but shouldn't the first tab usually be the default?
Probably more a smell of the design of this component, if it used a composition API we could conditionally render the tabs rather then having to fiddle with the data like this

Reply:
Yeah I'm okay with using first tab.